packages/manager/apps/web/client/app/domain/list/template.html

```diff
@@ -93,34 +93,19 @@
     </oui-datagrid-column>
 
     <oui-datagrid-column
-        data-title="::'domains_column_domains_transfer_state' | translate"
-        data-property="transferLockStatus"
-        data-type="options"
-        data-type-options="$ctrl.domainTransfertLockStatusColumnOptions"
-        data-filterable
-        data-sortable
-    >
-        <span
-            class="{{ 'oui-badge ' + $ctrl.DOMAINS_BADGES_TRANSFERT_LOCK_STATE[$value] }}"
-            data-translate="{{ 'domains_transfert_lock_status_' + $value }}"
-        ></span>
-        <button
-            data-ng-if="$value === $ctrl.DOMAIN_TRANSFERT_LOCK_STATE.LOCKED"
-            type="button"
-            class="oui-popover-button"
-            data-oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
-            data-oui-popover-placement="right"
-        ></button>
-        <button
-            data-ng-if="$value === $ctrl.DOMAIN_TRANSFERT_LOCK_STATE.UNLOCKED"
-            type="button"
-            class="oui-popover-button"
-            data-oui-popover="{{::'domains_transfert_lock_status_tooltip_' + $value | translate}}"
-            data-oui-popover-placement="right"
-        ></button>
-    </oui-datagrid-column>
+        data-title="::'domains_column_domains_renewal_mode' | translate"
+    ></oui-datagrid-column>
+
     <oui-datagrid-column
-        data-title="::'domains_column_domains_whois_owner' | translate"
+        data-title="::'domains_column_domains_renewal_date' | translate"
+    ></oui-datagrid-column>
+
+    <oui-datagrid-column
+        data-title="::'domains_column_domains_expiration_date' | translate"
+    ></oui-datagrid-column>
```

Comment:
Without `data-property`, the column will be empty

Reply:
It is normal : the columns completion are in other tickets, this is a just an initialization of the global structure of the datagrid.